assets/src/hooks/useVehicles.ts

```diff
@@ -116,10 +123,20 @@ const subscribe = (
   const handleVehicles = ({
     data: vehiclesAndGhostsData,
   }: {
-    data: VehicleOrGhostData[]
+    data: unknown
```

Comment:
This is the data that we get directly over the network, and by default it's given a type of `any` by the Phoenix channels library. However, we want it to be considered of type `unknown` so that it has to be refined by SuperStruct before being used, because `any` is actually compatible with any type, sort of the opposite of what we want. See the [introduction to `unkown`](https://www.typescriptlang.org/docs/handbook/2/functions.html#unknown).